components/omega/doc/userGuide/Eos.md

```diff
@@ -0,0 +1,15 @@
+(omega-user-eos)=
+
+## Eos
+
+The `Eos` class provides a container for the equation of state in OMEGA, and member functors that are the linear and TEOS-10 equations of state.
+Upon creation of an `Eos` instance, these functors are initialized and the array for the specific volume is allocated.
+The user-configurable options are: `EosType` (either `Linear' or `teos-10'), as well as the parameters needed for the linear equation of state.
```

Comment:
I think these are the wrong pairing of single quote mark

Reply:
I think maybe some references and short description for TEOS-10 could be useful to the user (you probably put this in the design doc, but might be worthwhile to repeat or at least link to here?). 

Also a description/name of the terms this routine outputs (with units maybe)? Kind of like the table in the tendency terms doc.